doc/releases/changelog-dev.md

```diff
@@ -273,6 +273,10 @@ keyword argument when using `GellMann`, which determines which of the 8 Gell-Man
 * `qml.tape.get_active_tape` is deprecated. Please use `qml.QueuingManager.active_context()` instead.
   [(#3068)](https://github.com/PennyLaneAI/pennylane/pull/3068)
 
+* `qml.Operator.compute_terms` is removed. On a specific instance of an operator, `op.terms()` can be used
```

Comment:
```suggestion
* `Operator.compute_terms` is removed. On a specific instance of an operator, `op.terms()` can be used
```
`Operator` isn't imported top-level. Just saying `Operator` should be fine, though it could also be called `qml.operation.Operator` if we want to include the full path.

Reply:
Thanks!